models/pathology_tumor_detection/docs/README.md

```diff
@@ -92,7 +101,15 @@ cd scripts && source evaluate_froc.sh
 
 #### Export checkpoint to TorchScript file
 
-TorchScript conversion is currently not supported.
```

Comment:
Hi @binliunls , could you please also help to remove:
https://github.com/Project-MONAI/model-zoo/blob/dev/ci/bundle_custom_data.py#L30